controllers/module_reconciler.go

```diff
@@ -95,6 +95,7 @@ func NewModuleReconciler(
 //+kubebuilder:rbac:groups=apps,resources=daemonsets,verbs=create;delete;get;list;patch;watch
 //+kubebuilder:rbac:groups="core",resources=nodes,verbs=get;list;watch
 //+kubebuilder:rbac:groups="core",resources=secrets,verbs=get;list;watch
+//+kubebuilder:rbac:groups="core",resources=configmaps,verbs=get;list
```

Comment:
Why do we need to list?
```suggestion
//+kubebuilder:rbac:groups="core",resources=configmaps,verbs=get
```

Reply:
just in case - we already have permission for get, maybe later we will use the list

Reply:
Then let us add it later, when we actually need it? 🙂

Reply:
fixed

Reply:
returning the list permissions: apparently, even if we use only Get API in the code, we still require the list permission, probably related to the caching